.binder/README.md

```diff
@@ -0,0 +1,31 @@
+# Docker Image for Binder
+
+<!--
+This is derived from the following setup:
+https://github.com/RobotLocomotion/drake/tree/dc2a9394d/.binder
+-->
+
+*Note that due to Binder conventions, this directory MUST always be in the root
+of the repository and named either `binder` or `.binder`. This image is NOT
+intended for use by most developers or users.*
+
+These instructions are for running the image locally. For Binder itself, you
+should only need to visit the link from the root-level README.
+
+To create a Docker image and run a Docker container similar to those used by
+[Binder](https://mybinder.org) for local debugging purposes, execute the
+following `build` and `run` commands from the top level of this Git repository:
+
+```bash
+docker build -f .binder/Dockerfile -t binder .
+docker run --rm -it --name mybinder -p 8888:8888 binder
+```
+
+For the URLs printed, only open the `127.0.0.1:8888` URL (including the login
+token) in a web browser on your host system.
+
+To stop the running container, simply exit it from the terminal with Ctrl+C.
+
+*Note*: If you want to test the Docker image with the current source tree
+(without copying, so you can modify source files), add the arguments
+`-v ${PWD}:/home/jovyan/pygccxml` to mount it directly.
```

Comment:
Does this need to be in double quotes (")? Additionally, I get the error response `docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"-v\": executable file not found in $PATH": unknown.`

Reply:
Nah, just needs to be reordered. If it's still confusing, it'd be best just to remove it, and leave it up to others to inspect `docker run` help.

 See usage string: `docker run [options...] IMAGE [COMMAND] [ARG...]`
It's because `-v` is for `docker`, but if you put it at the end, it's interpreted as `COMMAND` and overrides the entrypoint. Refer to docker docs for more info.

Reply:
Could you just clarify that this is for `docker-run`? Then we're good to merge!

Reply:
Done! edfa8d8